.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "root": true,
+  "plugins": ["@typescript-eslint"],
+  "parser": "@typescript-eslint/parser",
```

Comment:
Worth checking but you _might_ need to provide a [project](https://www.npmjs.com/package/@typescript-eslint/parser#user-content-parseroptionsproject) with optional [tsconfigRootDir](https://www.npmjs.com/package/@typescript-eslint/parser#tsconfigRootDir) to have the linting match type errors reported at build.

Reply:
Seems that this is only required for rules that require type information, I'm not using those for now (e.g. `plugin:@typescript-eslint/recommended-requiring-type-checking`)